src/main/java/bio/terra/cli/command/workspace/List.java

```diff
@@ -69,7 +69,7 @@ private void printText(java.util.List<UFWorkspaceLight> returnValue) {
 
   /** Column information for table output with `terra workspace list` */
   private enum Columns implements ColumnDefinition<UFWorkspaceLight> {
-    ID("ID", w -> w.id, 36, LEFT),
+    ID("ID", w -> w.id, 40, LEFT),
```

Comment:
Right now, this looks like

```
   ID                                        NAME                            GOOGLE PROJECT                  DESCRIPTION
   does-clone-copy-properties                (unset)                         terra-vdevel-snappy-beet-296    (unset)
   docker-demo                               (unset)                         terra-vdevel-cozy-pear-8873     (unset)
   1000-genomes                              1000 Genomes Phase 3            terra-vdevel-arctic-plum-2597   ## How to run notebook using 1000 Gen...
   20220907-creating-working-in-preprod-...  20220907 creating working i...  terra-vdevel-speedy-greens-681  (unset)
   private-data-collection                   Private Data Collection         terra-vdevel-smart-olive-258    This workspace simulates a private da...
   rnaseq-analysis-ak                        RNASeq Analysis AK              terra-vdevel-tepid-cashew-47    # Quality Check & Principal Component...
 ✓ buoyant-plum                              buoyant plum yo                 terra-vdevel-buoyant-plum-9341  (unset)
   af9c34ace-d32e-47df-af15-943259a528c3     dockerworkspace                 terra-vdevel-smart-mango-2125   hello this is a description
   emmarogge-test-ws                         emmarogge-test-ws               terra-vdevel-rapid-plum-8199    This workspace exists for testing TVC.
   emmarogge-test-ws-dsub                    emmarogge-test-ws-dsub          terra-vdevel-caring-apple-8479  This workspace provides examples of u...
   a66f7628b-aa58-4372-946c-25e9a15a6853     strong pear                     terra-vdevel-strong-pear-2651   (unset)
```

Is there an Azure equivalent to `GOOGLE PROJECT` column?

Do you want to hide the `GOOGLE_PROJECT` column if there are only Azure workspaces?

Reply:
I thought about this but wasn't sure, so didn't change anything yet. :)

For Azure we could show the subscription ID. So maybe:
* If there are only Azure workspaces, show `Subscription ID` column
* If there are only GCP workspaces, show `Google Project` column
* If there are both types of workspaces, show both columns

Reply:
SG. Could you write unit tests

// Check has `Azure Subscription` column and not `Google project`. (Don't need to check contents of column, just checking for heading is good enough)
listWorkspace_onlyAzure

etc